keyboards/ymd75/info.json

```
@@ -7,6 +7,94 @@
+                {"label":"7", "x":7, "y":1},
+                {"label":"8", "x":8, "y":1},
+                {"label":"9", "x":9, "y":1},
+                {"label":"0", "x":10, "y":1},
+                {"label":"_", "x":11, "y":1},
+                {"label":"+", "x":12, "y":1},
+                {"label":"Backspace", "x":13, "y":1, "w":2},
+                {"label":"Home", "x":15, "y":1},
+                {"label":"Tab", "x":0, "y":2, "w":1.5},
+                {"label":"Q", "x":1.5, "y":2},
+                {"label":"W", "x":2.5, "y":2},
+                {"label":"E", "x":3.5, "y":2},
+                {"label":"R", "x":4.5, "y":2},
+                {"label":"T", "x":5.5, "y":2},
+                {"label":"Y", "x":6.5, "y":2},
+                {"label":"U", "x":7.5, "y":2},
+                {"label":"I", "x":8.5, "y":2},
+                {"label":"O", "x":9.5, "y":2},
+                {"label":"P", "x":10.5, "y":2},
+                {"label":"{", "x":11.5, "y":2},
+                {"label":"}", "x":12.5, "y":2},
+                {"label":"Page Up", "x":15, "y":2},
+                {"label":"Caps Lock", "x":0, "y":3, "w":1.75},
+                {"label":"A", "x":1.75, "y":3},
+                {"label":"S", "x":2.75, "y":3},
+                {"label":"D", "x":3.75, "y":3},
+                {"label":"F", "x":4.75, "y":3},
+                {"label":"G", "x":5.75, "y":3},
+                {"label":"H", "x":6.75, "y":3},
+                {"label":"J", "x":7.75, "y":3},
+                {"label":"K", "x":8.75, "y":3},
+                {"label":"L", "x":9.75, "y":3},
+                {"label":":", "x":10.75, "y":3},
+                {"label":"@", "x":11.75, "y":3},
+                {"label":"~", "x":12.75, "y":3},
+                {"label":"Enter", "x":13.75, "y":2, "w":1.25, "h":2},
+                {"label":"Page Down", "x":15, "y":3},
+                {"label":"Shift", "x":0, "y":4, "w":1.25},
+                {"label":"|", "x":1.25, "y":4},
+                {"label":"Z", "x":2.25, "y":4},
+                {"label":"X", "x":3.25, "y":4},
+                {"label":"C", "x":4.25, "y":4},
+                {"label":"V", "x":5.25, "y":4},
+                {"label":"B", "x":6.25, "y":4},
+                {"label":"N", "x":7.25, "y":4},
+                {"label":"M", "x":8.25, "y":4},
+                {"label":"<", "x":9.25, "y":4},
+                {"label":">", "x":10.25, "y":4},
+                {"label":"?", "x":11.25, "y":4},
+                {"label":"Shift", "x":12.25, "y":4, "w":1.75},
+                {"label":"Up", "x":14, "y":4},
+                {"label":"End", "x":15, "y":4},
+                {"label":"Ctrl", "x":0, "y":5, "w":1.25},
+                {"label":"Win", "x":1.25, "y":5, "w":1.25},
+                {"label":"Alt", "x":2.5, "y":5, "w":1.25},
+                {"label":"Space", "x":3.75, "y":5, "w":6.25},
+                {"label":"AltGr", "x":10, "y":5, "w":1.5},
+                {"label":"Ctrl", "x":11.5, "y":5, "w":1.5},
```

Adding the extra right-hand mod key.

```suggestion
                {"label":"AltGr", "x":10, "y":5},
                {"label":"Menu", "x":11, "y":5},
                {"label":"Ctrl", "x":12, "y":5},
```